### ISSUE TITLE
Didn't get wlogout nor waybar to work by adding copying folders to ~/.config

### ISSUE THREAD

**Issue by thepenguinthatwants** [zDyanTB/HyprNova#8]
Hi zDyanTB!
I've tried to get wlogout and waybar to work just for testing purposes, but it didn't seem to do much.
I created folder ~/Pictures/wallpapers and put some wallpaper files there.

copied wlogout folder to .config/wlogout but it didn't' seem to work. Renamed nova.css to style.ss but everything has just white background.

Also added 
layerrule = blur, logout_dialog
to my hyprland.conf, but didn't help.

Any clue what  steps I might've missed?

**Comment by zDyanTB** [comment]
Hi bro, which way are you doing to show up wlogout?

**Comment by thepenguinthatwants** [comment]
Hi,
I've tried to just pick parts of it thinking that it would just work. 
Sorry, now I copied all the files from https://github.com/zDyanTB/HyprNova/tree/master/.config to my ~/.config folder. 

For some reason it's nagging about
hypr/configs/settings.conf at line 1: source? globbing error: found  no match
So assuming it's  missing
$HOME/.cache/wal/colors-hyprland.conf
So I created that folder and put file from https://github.com/zDyanTB/HyprNova/blob/master/.config/wal/templates/colors-hyprland.conf and it seems to still have same error. Any clue why it behaves like this?

Also tried to install necessary dependencies so it might work.
cliphist waybar swaync network-manager-applet wl-clipboard  hypridle swww wlogout.
But seems to be from something else.
